src/app/core/site-header/site-header.component.html

```diff
@@ -67,18 +67,24 @@
               data-bs-toggle="collapse"
               data-bs-target=".navbar-collapse.show"
             >
-              <span class="dropdown-item" (click)="useLanguage(lang.code)">
+              <span class="dropdown-item" (click)="useLanguage(lang.code); closeMenu()">
```

Comment:
to normalna praktyka? nie powinno closeMenu(); być odpalone wewnątrz useLanguage?

Reply:
@neu5 `useLanguage` powinno tylko zmieniać język; można by co najwyżej dodać nową metodę, która wykonuje te dwie operacje.

Reply:
A, to jest metoda tego komponentu, to może i tu może być w niej zamykanie, nie upieram się.